packages/core/src/DdSdkReactNative.tsx

```diff
@@ -176,6 +177,15 @@ export class DdSdkReactNative {
         return DdSdk.setTrackingConsent(consent);
     }
 
+    private static getSamplingStrategy = (
+        sampleRate: number
+    ): SessionSamplingStrategy => {
```

Comment:
any reason why you want to use strings instead of boolean here ?

Reply:
According to the RFC, a new value will be available later for delegation to the backend. 
I prefer to use union types (or enums if it is exposed for the SDK user) rather than booleans if there is a chance that it could be more than 2 values one day. 
That being said I could rename `YES` and `NO` by `SAMPLING_ENABLED` and `SAMPLING_DISABLED` to make it clearer once the delegation is implemented.